inspirehep/modules/migrator/tasks.py

```diff
@@ -288,9 +289,23 @@ def create_record(record):
     return json
 
 
+def update_documents_and_figures_for_record(record):
+    for attachment in chain(record.get('documents', []), record.get('figures', [])):
+        file_url = attachment.get('url')
+        filename = '{}_{}'.format(record.get('recid') + attachment.get('key'))
+        with file_resolving(file_url) as fp:
+            record.files[filename] = fp
+            record_file_obj = record.files[filename]
+            attachment['url'] = '/files/{bucket}/{key}'.format(
```

Comment:
Shouldn't this url start with /api ?

Reply:
This is already under api application cause is provided from invenio-files-rest...